src/components/topElements/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { IPurchase } from '../../utils/interfaces';
+import { getDate } from '../../utils/getDate';
+import './styles.css'
+
+interface TopElementsProps {
+    getPurchase: (arg0: IPurchase) => void;
+}
+
+const TopElements = ({getPurchase = (): void => {}}: TopElementsProps): JSX.Element => {
+    const [place, setPlace] = useState<string>('')
+    const [price, setPrice] = useState<number | undefined>()
+
+
+    const getNewPurchase = () => {
+        const date = getDate()
+        if (place && (price || price === 0)) {
+            getPurchase({place, price, date,  isEdit: false})
```

Comment:
please, call it addPurchase, because it's literally what it does, it adds a purchase, it doesn't get it or set it